src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -227,32 +228,32 @@ public boolean isDependentArgumentAllowed(final Class<?> predecessorClass) {
     @Override
     public void validateArguments() {
```

Comment:
Ok, but what's about a method in the argument collection for extra-validation of arguments, and call it at the end of this method?